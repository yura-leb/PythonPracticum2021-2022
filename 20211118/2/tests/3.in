class B:
    real = 'pabam '

class D:
    real = 'pabaaaaaaaam'

class F:
    real = 'pababababam'
    
class C:
    num = Num()

c, d, f = C(), C(), C()
c.num = B()
d.num = D()
f.num = F()
print(c.num)
print(c.num)
print(c.num+c.num + c.num + c.num + str(d.num))
print(f.num)
