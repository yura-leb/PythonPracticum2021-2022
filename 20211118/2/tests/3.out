pabam 
pabam 
pabam pabam pabam pabam pabaaaaaaaam
pababababam
